commands/create.go

```diff
@@ -42,29 +53,26 @@ func cmdCreate(c *cli.Context) {
 
 	certInfo := getCertPathInfo(c)
 
-	if err := setupCertificates(
-		certInfo.CaCertPath,
-		certInfo.CaKeyPath,
-		certInfo.ClientCertPath,
-		certInfo.ClientKeyPath); err != nil {
+	if err := setupCertificates(certInfo); err != nil {
 		log.Fatalf("Error generating certificates: %s", err)
 	}
 
-	defaultStore, err := getDefaultStore(
-		c.GlobalString("storage-path"),
-		certInfo.CaCertPath,
-		certInfo.CaKeyPath,
-	)
-	if err != nil {
-		log.Fatal(err)
+	store := &persist.Filestore{
+		Path:           c.GlobalString("storage-path"),
+		CaCertPath:     certInfo.CaCertPath,
+		PrivateKeyPath: certInfo.CaKeyPath,
 	}
 
-	provider, err := newProvider(defaultStore)
-	if err != nil {
-		log.Fatal(err)
+	switch driver {
```

Comment:
This doesn't do anything for now but essentially is intended to be the block where the driver options (strongly typed) are added in later on.